core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -1406,6 +1406,12 @@ public MetricKey build() {
               + "Use this metric to monitor the RPC pressure on worker.")
           .setMetricType(MetricType.GAUGE)
           .build();
+  public static final MetricKey WORKER_BLOCK_LOCK_POOL_REMAINING_RESOURCES_KEY =
```

Comment:
To be consistent with other metric keys
```suggestion
  public static final MetricKey WORKER_BLOCK_LOCK_POOL_REMAINING_RESOURCES_COUNT =
```

Reply:
sure thanks for flagging this.